gmplot/gmplot.py

```diff
@@ -269,12 +269,16 @@ def write_shapes(self, f):
             self.write_polygon(f, shape, settings)
 
     # TODO: Add support for mapTypeId: google.maps.MapTypeId.SATELLITE
-    def write_map(self,  f):
+    def write_map(self,  f, map_styles=None):
```

Comment:
We should avoid mutable objects in default function parameters:
http://docs.python-guide.org/en/latest/writing/gotchas/